bleak/backends/winrt/scanner.py

```diff
@@ -210,7 +210,7 @@ def _received_handler(
     def _stopped_handler(self, sender, e):
         logger.debug(
             "{0} devices found. Watcher status: {1}.".format(
-                len(self.seen_devices), self.watcher.status
+                len(self.seen_devices), "?" if self.watcher is None else self.watcher.status
```

Comment:
How about using `sender` instead?

```suggestion
                len(self.seen_devices), sender.status
```

Reply:
Sure, that's an option too!